elements/lisk-chain/src/account.ts

```diff
@@ -46,11 +47,13 @@ export class Account {
 	public username: string | null;
 	public isDelegate: number;
 	public nameExist: boolean;
```

Comment:
Do we need this? maybe we can remove this as well

Reply:
@ishantiw could you elaborate? I think that's used by delegate registration and not related to multi signatures right?

Reply:
Since its just a flag to check if `username` exists or not we can infer it from `username` field if it's `undefined` or not. I remember @shuse2 pointed it out in one of my PR and we decided to remove it later along with `multiMin` and `multiLifetime `

Reply:
There're 35 references to that field in the code. I think it's out of scope for this one. Can you create an issue to remove it in the future?